gene2phenotype_project/gene2phenotype_app/serializers/locus_genotype_disease.py

```diff
@@ -254,6 +256,49 @@ def get_date_created(self, id):
 
         return date
 
+    def get_curators(self, id):
+        """
+            List of curators who worked on the LGD record.
+            Dependency: this method depends on the history table.
+
+            Note: entries that were migrated from the old db have limited info details.
+        """
+        list_curators = set()
+        lgd_obj = self.instance
+        # Check LGD record history
+        history_records = lgd_obj.history.all().values('history_user__first_name', 'history_user__last_name')
```

Comment:
Suggestion - because we are going to also have changes to molecular mechanism (if undetermined ) and additional molecular_mechanism_evidence based on new publications, do we want to add a check for this?

Reply:
The endpoint to update the mechanism updates the history table `gene2phenotype_app_historicallocusgenotypedisease`.